src/js/collect/NavBar.jsx

```diff
@@ -134,6 +134,17 @@ export default function NavBar({ shiftPlotId, goToPlot, nextPlot, prevPlot, setP
                 icon="warning"
                 iconSize="20px"
                 onClick={() => setPlotAnswer("Mina")}
+                extraStyle={{ marginRight: "0.5rem" }}
+              />
+              <SvgButton
+                text={t("collect.clear")}
+                backgroundColor="var(--nav-bar-button)"
```

Comment:
Would be maybe nicer the background color or gray or same color as the boundary/label of a plot for the clear button

Reply:
Good idea. I'll update that